package.json

```diff
@@ -4,15 +4,16 @@
   "description": "Build CMS in few lines of code for different data sources",
```

Comment:
the package name should be removed.

Reply:
and description should move to `packages/canner/package.json`

Reply:
Removed and moved. Thanks!